app/src/test/java/org/simple/clinic/allpatientsinfacility/AllPatientsInFacilityLogicTest.kt

```diff
@@ -32,12 +33,28 @@ class AllPatientsInFacilityLogicTest {
       TrampolineSchedulersProvider()
   )
 
+  private lateinit var fixture: MobiusTestFixture<AllPatientsInFacilityModel, AllPatientsInFacilityEvent, AllPatientsInFacilityEffect>
+
   @Before
   fun setUp() {
     whenever(userSession.requireLoggedInUser())
         .thenReturn(Observable.just(user))
     whenever(facilityRepository.currentFacility(user))
         .thenReturn(Observable.just(facility))
+
+    fixture = MobiusTestFixture(
```

Comment:
Would it be possible to extract this into a separate `TestRule` that will automatically dispose the fixture? We could also try this out after we migrate another feature so we know what it needs to look like.

Reply:
Let's do this after we migrate another feature.

Reply:
Discussed offline: adding to the list of things to do.